bve/src/parse/mesh/errors.rs

```diff
@@ -1,16 +1,40 @@
+use crate::parse::Span;
+
+/// A single error in the parsing or evaluation of a mesh.
 #[derive(Debug, Clone, PartialEq)]
 pub struct MeshError {
+    /// Info about the exact error.
     pub kind: MeshErrorKind,
+    /// Location of the error within the file.
     pub span: Span,
 }
 
+/// Enum representing various types of errors encountered when parsing meshes
 #[derive(Debug, Clone, PartialEq)]
 pub enum MeshErrorKind {
-    UTF8 { column: Option<u64> },
+    /// Invalid UTF-8. May be Shift-JIS or other encoding.
+    UTF8 {
+        /// Column of the error. Only Optional due to the CSV library.
+        column: Option<u64>,
+    },
+    /// Index provided to vertex specific command is out of bounds.
     OutOfBounds { idx: usize },
-    DeprecatedInstruction { name: String },
-    UnknownInstruction { name: String },
-    GenericCSV { msg: String },
+    /// Instruction is deprecated in favor of another instruction
+    DeprecatedInstruction {
```

Comment:
Can you add a data member for what instruction I should use instead? Nothing grinds my gears more than finding out something is deprecated and not having any idea what to use instead

Reply:
Yeah they are just removed entirely as they don't do anything anymore. They can just be removed.

Reply:
Maybe `IgnoredInstruction` or `UnnecessaryInstruction` would be a better name